public/components/health-check/services/check-alerts-pattern.service.ts

```diff
@@ -0,0 +1,105 @@
+/*
+ * Wazuh app - Check alerts index pattern service
+ *
+ * Copyright (C) 2015-2021 Wazuh, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * Find more information about this on the LICENSE file.
+ *
+ */
+import { AppState, SavedObject } from '../../../react-services';
+import { getDataPlugin } from '../../../kibana-services';
+import { HEALTH_CHECK } from '../../../../common/constants';
+import { CheckLogger } from '../types/check_logger';
+
+export const checkAlerts = async (appConfig, checkLogger: CheckLogger) => {
```

Comment:
I think `checkIndexPatternObject` is a better name for this service and `check-index-pattern-object.service ` for the file

Reply:
Changed